test/JDBC/expected/BABEL-5129-vu-prepare.out

```diff
@@ -0,0 +1,28 @@
+
+-- Tests for ISNUMERIC function with varchar and nvarchar variables
+CREATE TABLE babel_5129 (
+    int_type int,
```

Comment:
Please add tests for `bigint, smallint, tinyint, bit, float, real` datatypes, as well as special characters that should return 1 such as `$, +, -`

Reply:
(If this is already tested in another file please add these examples to the description)